docs/docs/vulnerability/examples/cache.md

```diff
@@ -53,4 +76,4 @@ $ trivy server --cache-backend redis://localhost:6379 \
   --redis-key /path/to/key.pem
 ```
 
-TLS option for redis is hidden from Trivy command-line flag, but you still can use it.
\ No newline at end of file
+TLS option for redis is hidden from Trivy command-line flag, but you still can use it.
```

Comment:
The TLS option for Redis is hidden from the Trivy command-line flag, but you still can use it.

- make sure to use Redis uppercase everywhere :)